Orange/widgets/visualize/owmosaic.py

```diff
@@ -395,7 +404,7 @@ def set_data(self, data):
 
     def init_vizrank(self):
         if self.data is not None and len(self.data) > 1 \
-                and len(self.data.domain.attributes) >= 1:
+                and len(self.data.domain.attributes) >= 2:
```

Comment:
The condition should be applied to `len(self.data.domain.variables + self.data.domain.metas)`. (The mosaic is build upon all variables in the domain).